runtests.sh

```diff
@@ -23,19 +32,27 @@ export PYTHONPATH=`pwd`/src
 echo "Create Virtualenv for Python deps ..."
 function prepare_venv() {
     VIRTUALENV=`which virtualenv`
-    if [ $? -eq 1 ]; then
+    if [ $? -eq 1 ]
+    then
         # python34 which is in CentOS does not have virtualenv binary
         VIRTUALENV=`which virtualenv-3`
     fi
 
     ${VIRTUALENV} -p python3 venv && source venv/bin/activate
+    if [ $? -ne 0 ]
+    then
+        printf "${RED}Python virtual environment can't be initialized${NORMAL}"
+        exit 1
+    fi
     pip install -U pip
```

Comment:
It's `pip` or `pip3` here?

Reply:
it is correct, because the `pip` now surely runs in the virtual environment:

```
$ which pip
/usr/bin/pip

$ pip --version
pip 9.0.1 from /usr/lib/python2.7/site-packages (python 2.7)

$ virtualenv -p python3 venv && source venv/bin/activate
Running virtualenv with interpreter /usr/bin/python3
Using base prefix '/usr'
New python executable in /home/ptisnovs/venv/bin/python3
Also creating executable in /home/ptisnovs/venv/bin/python
Installing setuptools, pip, wheel...done.

$ which pip
~/venv/bin/pip

$ pip --version
pip 10.0.1 from /home/tester/venv/lib/python3.6/site-packages/pip (python 3.6)
```

Reply:
let's check it :)